src/components/video-texture-target.js

```diff
@@ -64,38 +132,56 @@ AFRAME.registerComponent("video-texture-target", {
         texture.minFilter = THREE.LinearFilter;
         texture.encoding = THREE.sRGBEncoding;
 
-        // Copy texture settings from the original texture so that things like texture wrap settings are applied
-        const originalTexture = this.originalMap || this.originalEmissiveMap;
-
-        if (originalTexture) {
-          texture.wrapS = originalTexture.wrapS;
-          texture.wrapT = originalTexture.wrapT;
+        this.applyTexture(texture);
+      });
+    } else {
+      if (this.data.target) {
+        const videoTextureSource = this.data.target.components["video-texture-source"];
+        const texture = videoTextureSource.renderTarget.texture;
+        this.applyTexture(texture);
+
+        // Bit of a hack here to only update the renderTarget when the screens are in view
+        material.map.isVideoTexture = true;
+        material.map.update = () => {
+          videoTextureSource.textureNeedsUpdate = true;
+        };
+      } else {
+        if (material.map && material.map !== this.originalMap) {
+          disposeTexture(material.map);
```

Comment:
If I'm reading this right, this will `dispose` a `video-texture-source`'s `renderTarget.texture`. It seems like that should be the responsibility of the `video-texture-source`. 

Related, if you have two jumbotron `video-texture-targets` with the same `video-texture-source`, and then you delete or change the `src / target` of one of the jumbotrons, will the second jumbotron get messed up?

Reply:
Ah yeah, didn't re-review this code yet, just carried it over from the existing code... I think the `video-'texture-source` should be responsible for cleaning this up. Will probably rework the video avatar stuff so that it just ends up passing in a texture so it can be the one managing its own texture as well.